bot/exts/moderation/modpings.py

```diff
@@ -104,7 +104,9 @@ async def off_command(self, ctx: Context, duration: Expiry) -> None:
             self._role_scheduler.cancel(mod.id)
         self._role_scheduler.schedule_at(duration, mod.id, self.reapply_role(mod))
 
-        await ctx.send(f"{Emojis.check_mark} Moderators role has been removed until {until_date}.")
+        embed = Embed(timestamp=duration, colour=Colours.bright_green)
+        embed.set_footer(text="Moderators role has been removed", icon_url=Icons.green_checkmark)
```

Comment:
```suggestion
        embed.set_footer(text="Moderators role has been removed until", icon_url=Icons.green_checkmark)
```
I'm not sure what other's think, but I preferred the wording you have before, with `until` here. It adds context to the date.

Reply:
We could also move the current footer to the embed's text, and just have "Until" in the footer.

Reply:
> I'm not sure what other's think, but I preferred the wording you have before, with `until` here. It adds context to the date.

I removed it by accident, my bad.

Reply:
Just adding back until seems ok, on second thought

Reply:
Resolved in https://github.com/python-discord/bot/pull/1556/commits/5c1cf2b5fd94359d42d147ce8687dd27b05e193d.